src/controllers/factorycontroller.js

```diff
@@ -204,10 +193,23 @@ export default class FactoryController extends OrderController
                         let formValues = $("form[name=newFactoryOrder]").serializeArray();
                         let products = OrderController._makeOrder(formValues);
 
-                        $("#factory-order-cost").html(products.reduce((sum, prod) => sum + prod.stockValue(), 0));
+                        $("#factory-order-cost").html(products.reduce((sum, prod) => sum + prod.stockValue(), 0).toFixed(2));

```

Comment:
Does this work now?

Reply:
Yes! There was again some redundant code that I was looking at.